connector/src/test/resources/application.properties

```diff
@@ -26,7 +26,7 @@ application.usagecontrol.constraint.location=EU
 application.usagecontrol.constraint.purpose=demo
 #spring.data.mongodb.authentication-database=admin
 spring.data.mongodb.host=localhost
-spring.data.mongodb.port=27018
+spring.data.mongodb.port=27017
```

Comment:
Leftover from some testing? Why real port or it is not needed, since TestContainers will override dockerized port for mongo?

Reply:
Probably leftover from some testing, but as you mentioned it should affect anything since it will be overriden.